plugins/available/less-pretty-cat.plugin.bash

```diff
@@ -13,9 +13,12 @@ if $(command -v pygmentize &> /dev/null) ; then
       about 'runs either pygmentize or cat on each file passed in'
       param '*: files to concatenate (as normally passed to cat)'
       example 'cat mysite/manage.py dir/text-file.txt'
+      if [ -z "$BASH_IT_CCAT_STYLE" ]; then
```

Comment:
This could be simplified into a one line statement:

```bash
BASH_IT_CCAT_STYLE="${BASH_IT_CCAT_STYLE:=default}"
```
I also recommend to put this at the top of the file, outside of the functions, which will only execute this once, not every time you call the function.

Reply:
done